src/app/settings/views/Configuration/Commissioning/Commissioning.test.tsx

```diff
@@ -58,14 +58,14 @@ describe("Commissioning", () => {
       </Provider>
     );
 
-    expect(wrapper.find("Spinner").exists()).toBe(true);
+    expect(screen.getByText("Loading...")).toBeInTheDocument();
```

Comment:
It would be good to use a Label enum here too.

Reply:
Added 👍